linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers_ordered_stig/oval/shared.xml

```diff
@@ -31,7 +31,7 @@
 
   <ind:textfilecontent54_object id="obj_sshd_use_approved_ciphers_ordered_stig" version="1">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*(?i)Ciphers(?-i)[\s]+(?=[\w]+)(aes256-ctr(?=[\w,]+|$),?)?(aes192-ctr(?=[\w,]+|$),?)?(aes128-ctr)?[\s]*(?:#.*)?$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*[Cc]iphers[\s]+(?=\w)(aes256-ctr(?=[\w,-@]+|$),?)?(aes192-ctr(?=[\w,-@]+|$),?)?(aes128-ctr(?=[\w,-@]+|$),?)?[\s]*(?:#.*)?$</ind:pattern>
```

Comment:
From: https://man7.org/linux/man-pages/man5/sshd_config.5.html
```
The possible keywords and their meanings are as follows (note that
     keywords are case-insensitive and arguments are case-sensitive):
```

It means that `CIpHeRS` should also be accepted for example.

Also your regex doesn't match a line with trailing whitespace. I don't know exactly why but the previous one accepted it

I use this website to test regexes: https://regex101.com/

Reply:
Can you explain what's the intent behind these changes? Thank you

Reply:
I've got a use case where I need to adhere more closely to DISA (after remediation, I expect DISA's upstream rules to be evaluated on the system by a separate auditor), and I was looking for a path of less resistance.

Reply:
I believe the original `(?i)Ciphers(?-i)` should be enough to cover the case-insensitivity, although I don't really believe that people will use `CIpHeRs` in their SSH configuration files.